static/js/stream_create.js

```diff
@@ -16,6 +16,27 @@ exports.get_name = function () {
     return created_stream;
 };
 
+var stream_subscription_error = (function () {
+    var self = {};
+
+    self.report_no_subs_to_stream = function () {
+        $("#stream_subscription_error").text(i18n.t("Stream must have at least one subscriber."));
```

Comment:
Similar comment to the next one; this should probably be "You cannot create a stream with no subscribers!"